### PR TITLE
MOTECH-2617: Fixed a condition which wasn't saving a first configuration as default.

### PR DESCRIPTION
**Pull request by mkruszynski**
…set as default

### CODE REVIEW

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

File: commcare/src/main/resources/webapp/js/controllers.js [hunk 1]
```diff
@@ -554,7 +554,7 @@
                     $scope.verifySuccessMessage = $scope.msg('commcare.save.success');
                     $scope.verifyErrorMessage = '';
                     $scope.connectionVerified = true;
-                    if ($scope.configurations.defaultConfigName === "") {
+                    if ($scope.configurations.defaultConfigName === "" || $scope.configurations.defaultConfigName === null) {
```
Comment (pgesek) [inline commcare/src/main/resources/webapp/js/controllers.js]: just do:  if (!$scope.configurations.defaultConfigName)

**Comment by sebbrudzinski** [comment]
test this please

**Pull request merged**